Noti/PushManager.swift

```diff
@@ -459,6 +431,76 @@ class PushManager: NSObject, WebSocketDelegate, NSUserNotificationCenterDelegate
         
     }
     
+    // Given a push JSON object from Pushbullet create a notification
+    // Used in mirror and subtipe:push
+    internal func createNotification(push : JSON) -> NSUserNotification{
+        let notification = NSUserNotification()
+        notification.actionButtonTitle = "Show"
+        if let url = push["url"].string {                       // Special type of Noti: url type
```

Comment:
Yep! Next commit I'll clean the code! 
Can I correct also all the JSON.parse() calls? It gives me 10 warning that is deprecated.